src/core/tritonserver.h

```diff
@@ -1288,6 +1295,27 @@ TRITONSERVER_EXPORT TRITONSERVER_Error* TRITONSERVER_ServerModelBatch(
     TRITONSERVER_Server* server, const char* model_name,
     const int64_t model_version, uint32_t* flags, void** voidp);
 
+/// Get the transaction policy of the model. The policy is communicated
+/// by a flags value.
+///
+///   - TRITONSERVER_TXN_POLICY_ONE_TO_ONE: The model generates exactly
+///     one request per response.
+///
+///   - TRITONSERVER_TXN_POLICY_DECOUPLED: The model may generate multiple
+///     responses per request.
+///
+/// \param server The inference server object.
+/// \param model_name The name of the model.
+/// \param model_version The version of the model.  If -1 then the
+/// server will choose a version based on the model's policy.
+/// \param txn_flags Returns flags indicating the transaction policy of the
+/// model.
+/// \return a TRITONSERVER_Error indicating success or failure.
+TRITONSERVER_EXPORT TRITONSERVER_Error*
+TRITONSERVER_ServerModelTransactionPolicy(
```

Comment:
Let's call this ...TransactionProperties(). I'm going to change the batch properties name to TRITONSERVER_ServerModelBatchProperties() as well. Can you also add a voidp argument as in ModelBatch to provide some future proofing.